Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -329,11 +329,11 @@ def _parse_qresult(self):
                     if hit:
                         # need to keep track of hit IDs, since there could be duplicates,
                         if hit.id in key_list:
-                            warnings.warn("Adding hit with BLAST-generated ID "
-                                    "%r since hit ID %r is already present "
-                                    "in query %r. Your BLAST database may contain "
+                            warnings.warn("Renaming hit ID %r to a BLAST-generated ID "
```

Comment:
Really minor change, but I'd remove ``a`` here, and there should only be one BLAST generated ID which could be used?